### ISSUE TITLE
Example usage of webpack import

### ISSUE THREAD

**Issue by oliverswitzer** [peermetrics/webrtc-stats#11]
Hi,

I noticed that the example in the example directory uses a script tag to import the JS... is there a way to import this library via webpack? 

I've tried the following: 

```.js
import * as WebRTCStats from '@peermetrics/webrtc-stats'
import {WebRTCStats} from '@peermetrics/webrtc-stats'
import WebRTCStats from '@peermetrics/webrtc-stats'
```

None of these options seem to work when I try to construct the object...

```.js
let webrtcStatsMonitor = new WebRTCStats({
  getStatsInterval: 5000
})
```

In the console I get

```
Uncaught TypeError: (_peermetrics_webrtc_stats__WEBPACK_IMPORTED_MODULE_11___namespace_cache || (intermediate value)(intermediate value)) is not a constructor
```

It seems as though the imported JS returns an object like so: 

```.js
{ 
  window: { WebRTCStats: class WebRTCStats }
}
```

Any help would be much appreciated.

Thanks!

**Comment by onel** [comment]
Hey there, sorry for the problem. 
I haven't tested with webpack but using a normal import should work without problems. For a normal import this is the right way to do it:
```
import {WebRTCStats} from '@peermetrics/webrtc-stats'
```
It might be related to the fact that in `package.json` we have a `main` and a `browser` entry.
Let me check this and get back to you but thanks for raising this issue.

Another option would be to load the lib with a script tag but I know that's not ideal.

**Comment by onel** [comment]
It seem that the `browser` entry in package.json was the main culprit. That bundle is intended to be used directly in the browser and not inside a bundler, like webpack.
This issue is fixed in the new version, `3.1.1`.

To import the lib, use
```
import {WebRTCStats} from '@peermetrics/webrtc-stats'
```

Thanks again for raising this issue.

**Comment by oliverswitzer** [comment]
@onel amazing thank you for the quick turnaround on this! Just validated that this now works 👍
